test-pom/pom.xml

```diff
@@ -43,6 +43,16 @@ THE SOFTWARE.
     <jacocoSurefireArgs /><!-- empty by default -->
   </properties>
 
+  <dependencyManagement>
```

Comment:
Not sure it is needed after spotbugs-annotations , but I do not worry much about the test code

Reply:
> Not sure it is needed

It is needed, for the reasons I explained in the PR description (see the section starting with "The only other issue I had was…").